subjects/gaming/firing-range/README.md

```diff
@@ -1,39 +1,125 @@
-## FiringRange
+## Firing-range
```

Comment:
i see that different formats is used in the gaming project and the one that u chosed should be apply to all the other project.

Reply:
yes, i will be using kebab case in the project repos, but the names inside the subject file sometimes look weird when applying kebab case thats why i go for normal text, like 2.5D Adventure (2-5-d-adventure in kebab case)

Reply:
Just make sure to keep it consistent over the branch 👍